aws/resource_aws_lb_test.go

```diff
@@ -941,6 +992,51 @@ resource "aws_subnet" "alb_test" {
 `, lbName)
 }
 
+func testAccAWSLBConfig_networkLoadbalancerAccessLogs(lbName string) string {
+	return fmt.Sprintf(`resource "aws_lb" "lb_test" {
+  name            = "%s"
+  internal        = true
+  load_balancer_type = "network"
+
+  enable_deletion_protection = false
+
+  access_logs {
+    bucket = "${aws_s3_bucket.logs.bucket}"
```

Comment:
This line and the one after will trigger issues: 
* no such `var.bucket_prefix` variable
* no such logs bucket

Could you have a look? otherwise looks good to me!